scala-package/core/pom.xml

```diff
@@ -138,6 +138,11 @@
       <version>INTERNAL</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
Since we have commons-codec and commons-io defined in the parent pom, it may not be necessary to redefine in here.

Reply:
commons-codec is defined in core for the apidoc-generation stage.
commons-io can be removed. I'll do that.